py/visdom/server.py

```diff
@@ -930,6 +979,8 @@ def download_scripts(proxies=None, install_dir=None):
         # here is another url in case the cdn breaks down again.
         # https://raw.githubusercontent.com/plotly/plotly.js/master/dist/plotly.min.js
         'https://cdn.plot.ly/plotly-latest.min.js': 'plotly-plotly.min.js',
+        # Stanford Javascript Crypto Library for Password Hashing
+        '%ssjcl@1.0.7/sjcl.js' % b: 'sjcl.js',
```

Comment:
Likely the last comment - can you add this file to our `download.sh` script, so that those who download with the script will be able to get it as well?

Reply:
done @JackUrb